### PR TITLE
Implement LoRa sending + add a separate fiel for the receiver

### PR DESCRIPTION
**Pull request by anarchuser**
Note:
* SD card reader and LoRa module didn't work together
* The receiver currently only prints the received values to serial, nothing more

### CODE REVIEW

**Comment by Urhengulas** [comment]
Can please get the CI to compile and resolve the merge conflict?

File: main/main.ino [hunk 1]
```diff
@@ -44,6 +64,13 @@ float last_height;
 void loop()
 {
   update_sensors();
+  digitalWrite(PB9, HIGH);
+  delay(800);
+  digitalWrite(PB9, LOW);
+  delay(200);
```
Comment (Urhengulas) [inline main/main.ino]: Do we want to have that long of a delay in the main loop? In the past we had ~15 control cycles per second, with this less than one.

File: networking/rocket.capnp [hunk 2]
```diff
@@ -0,0 +1,26 @@
+interface Rocket {
```
Comment (Urhengulas) [inline networking/rocket.capnp]: What is this file for?

**Review by Urhengulas — CHANGES_REQUESTED** [review]
If you could rebase it, you will also fix the CI error, since there the new library is already specified.

File: main/main.ino (hunk 1 above)
Comment (anarchuser) [reply main/main.ino]: ah no, we only needed that for debugging

File: networking/rocket.capnp (hunk 2 above)
Comment (anarchuser) [reply networking/rocket.capnp]: it's irrelevant, we used it for demonstration purposes for the Network module

**Pull request closed without merging**